html/mix-manifest.json

```diff
@@ -1,12 +1,8 @@
 {
-    "/js/app.js": "/js/app.js?id=d074dd82ac08dba78c44",
```

Comment:
This seems wrong

Reply:
@Jellyfrog 
i ran this command `lnms translation:generate` and this diff from mix-manifest.json is the result
Maybe i miss something?

Reply:
works for me, did you do "npm install" before?

Reply:
ok i did `npm install` and runs lnms translation again now in diff only

vendor.js and <language>.js is changed

should it be like this? If so, i'll commit it :-)

Reply:
@Jellyfrog fixed it .. hope so :-)